io.openems.edge.io.revpi/src/io/openems/edge/io/revpi/Config.java

```diff
@@ -14,8 +14,11 @@
 	@AttributeDefinition(name = "Alias", description = "Human-readable name of this Component; defaults to Component-ID")
 	String alias() default "";
 
-	@AttributeDefinition(name = "Is enabled?", description = "Is this Component enabled?")
+	@AttributeDefinition(name = "Enabled", description = "Enable Component")
```

Comment:
The former was really the standard that we use everywhere. Is there a reason why you changed it?

See e.g. https://github.com/OpenEMS/openems/blob/develop/io.openems.edge.batteryinverter.kaco.blueplanetgridsave/src/io/openems/edge/batteryinverter/kaco/blueplanetgridsave/Config.java#L19

Reply:
Sorry, no reason why I changed this. I switched it back to the standard.